api/rpc/account/mock.go

```diff
@@ -0,0 +1,52 @@
+package account
+
+import "github.com/appcelerator/amp/data/schema"
+
+var accounts = []*schema.Account{
+	{
+		Id:         "0",
+		Name:       "generalhenry",
+		Type:       schema.AccountType_USER,
+		Email:      "hallentilford@axway.com",
+		PwHashcode: hash,
```

Comment:
I should have commented before, but let's refactor the field name (it's so idiomatic to just say password hash): PwHashCode -> `PasswordHash`.